aqua/pngpaste/Portfile

```
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                pngpaste
+version             0.2.3
+categories          aqua
+platforms           darwin
+license             BSD
+maintainers         telotortium
+description         paste PNG files
+long_description    Paste PNG into files on MacOS, much like pbpaste does for text.
+homepage            https://github.com/jcsalterego/pngpaste
+master_sites        https://github.com/jcsalterego/pngpaste
+
+github.setup        jcsalterego pngpaste 0.2.3
+fetch.type          git
+
+use_configure       no
+destroot             {
+    xinstall -m 755 -d ${destroot}${prefix}/bin
+    xinstall -m 755 ${worksrcpath}/pngpaste ${destroot}${prefix}/bin
+}
```

```suggestion
use_configure       no
destroot            {
    xinstall ${worksrcpath}/pngpaste ${destroot}${prefix}/bin
}
```

I think we can simplify this a bit. The default mode is `755` so it's already set, and `${destroot}${prefix}/bin` should already be created.

See https://guide.macports.org/#reference.tcl-extensions for more info.

nitpick: I also aligned the `{` to match the spacing.